### ISSUE TITLE
[Proposal] Modifications for ScumVM

### ISSUE THREAD

**Issue by Skeeve** [cbepx-me/RG35xx-P-RG35xx-H-Modification#39]
Please see this reddit post: https://www.reddit.com/r/RG35XX_H/comments/1de1u50/make_scummvm_look_more_like_the_other_emulators/

**Comment by TheKerevas** [comment]
I agree, I tried it and it really works, very easy and quick to add, I hope it is added natively on stock mod!

**Comment by TheKerevas** [comment]
It seems I have integrated your request, I formatted the SD and installed the system completely and found the change already inserted

**Comment by Skeeve** [comment]
That's weird as it's not implemented in 2.5

Anyhow: I developed a very new version which requires just ONE LINE to be added to RA_Launch.sh.

**Comment by TheKerevas** [comment]
> That's weird as it's not implemented in 2.5
> 
> Anyhow: I developed a very new version which requires just ONE LINE to be added to RA_Launch.sh.

And how can I then see the games like when I modified RA_Launch.sh? As I said, I formatted and redid the SD 😱

**Comment by Skeeve** [comment]
I have not the slightest clue, as I can look neither onto your card nor onto your device.

**Comment by Skeeve** [comment]
As I have found a better solution which is compatible to the standard AND supports hiding directories, and is compatible to every emulator core, I will close this issue now and (later) open a new proposal.
